src/generate.ts

```diff
@@ -55,16 +56,18 @@ const outputPath = path.resolve(argv.output);
 export async function run(): Promise<void> {
   const templateUrl = (templates as { [name: string]: string })[argv.template];
 
-  // clone selected template
-  console.info(`Setting up template: ${argv.template}`);
-  if (fs.existsSync(templatePath)) {
+  // download selected template
+  if (fs.pathExistsSync(templatePath)) {
```

Comment:
so this is returning a falsy value?

Reply:
It's actually a boolean, it does the same as `existsSync`, I just wanted to use the `fs-extra` syntax